src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -171,17 +190,19 @@ def get_vm_size_completion_list(prefix, action, parsed_args, **kwargs):  # pylin
 
     register_cli_argument(scope, 'os_disk_name', help='The name of the new VM OS disk.', arg_group='Storage')
     register_cli_argument(scope, 'os_type', help='Type of OS installed on a custom VHD. Do not use when specifiying an URN or URN alias.', arg_group='Storage', **enum_choice_list(['windows', 'linux']))
-    register_cli_argument(scope, 'storage_account', help='The name to use when creating a new storage account or referencing an existing one. If omitted, an appropriate storage account in the same resource group and location will be used, or a new one will be created.', arg_group='Storage')
+    register_cli_argument(scope, 'storage_account', help="Only applicable when use with '--use-unmanaged-disk'. The name to use when creating a new storage account or referencing an existing one. If omitted, an appropriate storage account in the same resource group and location will be used, or a new one will be created.", arg_group='Storage')
```

Comment:
I will clarify with CRP folks. Like i explained `unmanaged` in a av-set means a different thing